bgpd/bgp_fsm.h

```diff
@@ -56,6 +56,50 @@
 #define FSM_PEER_TRANSFERRED    2
 #define FSM_PEER_TRANSITIONED   3
 
+#define BGP_PEER_GR_HELPER_ENABLE(peer)	\
+	do {		\
+		UNSET_FLAG( \
```

Comment:
curious: why unset is needed here?

Reply:
Unset is need for the scenario, let say when a peer is transiting from Graceful Restart to only Helper Mode.